source/browseMode.py

```diff
@@ -2036,21 +2036,30 @@ def clearAppSelection(self):
 	)
 	def script_toggleNativeAppSelectionMode(self, gesture: inputCore.InputGesture):
 		if not self._nativeAppSelectionModeSupported:
-			# Translators: the message when native selection mode is not available in this browse mode document.
-			ui.message(_("Native selection mode unsupported in this document"))
+			if not self._nativeAppSelectionMode:
+				# Translators: the message when native selection mode is not available in this browse mode document.
+				ui.message(_("Native selection mode unsupported in this browse mode document"))
+			else:
+				# Translators: the message when native selection mode cannot be turned off in this browse mode document.
+				ui.message(_("Native selection mode cannot be turned off in this browse mode document"))

```

Comment:
"browse mode document" -> "document"

My feeling is that, for users and translators, there is no need to specify here that the document is a browse mode document (no possible ambiguity). This way, the messages are lighter. Feel free to ignore this suggestion if you disagree with me though.


```suggestion
				# Translators: the message when native selection mode is not available in this document.
				ui.message(_("Native selection mode unsupported in this document"))
			else:
				# Translators: the message when native selection mode cannot be turned off in this document.
				ui.message(_("Native selection mode cannot be turned off in this document"))
```